ui/raidboss/data/06-ew/raid/p6s.ts

```diff
@@ -85,6 +102,235 @@ const triggerSet: TriggerSet<Data> = {
+          return;
+
+        const poly6Map: { [l: string]: string } = {
+          insideNW: 'insideSE',
+          insideNE: 'insideSW',
+          insideSE: 'insideNW',
+          insideSW: 'insideNE',
+        };
+
+        switch (data.polyInstance) {
+          case 1: // four safe spots: two inside, two outside.  call only the inside ones.
+            if (safeTiles.length !== 4 || safe1 === undefined || output[safe0] === undefined || output[safe1] === undefined)
+              return;
+            return output.combo!({ dir1: output[safe0]!(), dir2: output[safe1]!() });
+          case 2: // one inside safe spot
+            if (safeTiles.length !== 1 || output[safe0] === undefined)
+              return;
+            return output.single!({ dir1: output[safe0]!() });
+          case 3: // two inside safe spots
+            if (safeTiles.length !== 2 || safe1 === undefined || output[safe0] === undefined || output[safe1] === undefined)
+              return;
+            return output.combo!({ dir1: output[safe0]!(), dir2: output[safe1]!() });
+          case 4: // lots of safe spots, so give generic warning
+            return output.polyAvoid!();
+          case 5: // two outside safe spots (reduced to one by Chorus Ixou)
+            if (safeTiles.length !== 2 || safe1 === undefined)
+              return;
+            if ((outsideFrontBackTiles.includes(safe0)) && (outsideSideTiles.includes(safe1))) { // should be always true because of ordering
+              data.poly5FrontBackTile = safe0;
+              data.poly5SideTile = safe1;
+              return; // success - output will be handled by Chorus Ixou trigger
+            }
+            return output.polyAvoid!();
+          case 6: // Cachexia 2 - four safe spots that form corners of a 3x3 tile sub-grid, so just call the center
+            if (safeTiles.length !== 4 || poly6Map[safe0] === undefined)
+              return;
+            return output.polyCachexia!({ dir1: output[poly6Map[safe0]!]!() });
+          case 7: // one inside safe spot
+            if (safeTiles.length !== 1 || output[safe0] === undefined)
+              return;
+            return output.single!({ dir1: output[safe0]!() });
+          case 8: // four safe spots, two inside two outside.  call only the inside ones.
+            if (safeTiles.length !== 4 || safe1 === undefined || output[safe0] === undefined || output[safe1] === undefined)
+              return;
+            return output.combo!({ dir1: output[safe0]!(), dir2: output[safe1]!() });
+          default:
+            return;
+        }
+      },
+      run: (data) => {
+        data.tileTethers = [];
+        data.mapEffects = [];
+        data.combatantData = [];
+      },
+      outputStrings: {
+        combo: {
+          en: 'Safe Tiles: ${dir1} / ${dir2}',
+        },
+        single: {
+          en: 'Safe Tile: ${dir1}',
+        },
+        polyCachexia: {
+          en: 'Safe Tiles: Intercards of ${dir1}',
```

Comment:
I understand what you're getting at here, but I personally find this wording very hard to parse.  I certainly could not call this out and have anybody understand it.  Maybe this is just me?

I was wondering it'd make sense to just say "inside east" or "outside west" depending on which debuff you had, since at least being in the right column is the most important part here, and it's pretty easy to figure out where to go from there.

Reply:
> I understand what you're getting at here, but I personally find this wording very hard to parse. I certainly could not call this out and have anybody understand it. Maybe this is just me?

To be honest, it's a pretty difficult mechanic to call, because there are four safe squares that are non-adjacent, and each needs to be utilized by a different person/group to successfully resolve the mechanic.  Combine that with the fact that there are different strats (aoe N stack S vs. aoe S stack N), and there's a lot of room for confusion.

I hesitate though about doing "inside east"/"outside west" though, because we use the "inside x / outside x" nomenclature in p8s to call adjacent pairs of safe tiles (and now will be doing the same with poly 1 & 8), so i think that risks even more confusion.  And at the end of the day, being in the wrong tile, even if on the right side, is at least a damage down if not death.

Two other options to consider.  Since we do know which side (e/w) the player needs to be on, we could call it as:
1. Go North/South of ${dir1} (and have $dir1 be the tile in between the N/S safe tiles on the correct side.  This might be more clear.  Downside is that it's not a call that you could call out universally for the entire raid, as it will only apply to 4 people.
2. Call the two safe tiles directly (${dir1} and ${dir2}) on the safe side.  This is even more direct, but (a) it means using "Outside WNW" (or whatever) as part of the call, and personally, I hate how nasty that is, and (b) same problem as above in only applying to 4 people.

Just not sure the right way to concisely communicate the info here, both for the individual to resolve and for people doing callouts.

Reply:
3. I suppose the third option here (which is what one of my groups does) is to call the inside safe tile (there will only be one) as the "anchor tile", and everyone knows the rest of the safe tiles based on that (inside tile, opposite corner, and then the other two).  But again, not sure how to concisely communicate that in a call.

Reply:
Ok, sleep + caffeine + more thought, and here's (yet another) option but I think it's the best of all worlds.

- infoText (~8:52) when the mapeffect lines are sent that calls the inside safe tile and the corner safe tile (e.g. "Safe Tiles: Inside NE / Corner SW".
- alertText (~8:57) when the two stack markers & two aoe markers appear that calls out the side the player should be on and whether they should stack or spread based on what headmarker they get (or none)  (e.g. "Stack East" or "Spread West")
- cach2 resolves (~9:02)

I like this approach because it calls the just enough of the safe tiles, and early enough, for players to find at least one safe tile on the correct side, and it then gives a player-specific instruction to allow them to resolve whether they should be in the "north safe" or "south safe" tile on their east/west side (based on whatever strat their group is doing).  What do you think?

Reply:
You know at 8:52 which side you're on, so it could just call the safe spots on your side, e.g. `Safe Tiles: Inside NE, Outside SSE`? Otherwise I think this is good.

Reply:
Works for me.